.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: Release
+
+on:
+  release:
+    types:
+    - published
+  schedule:
+  - cron: '0 23 * * *'
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Release
+    uses: OpenTTD/actions/.github/workflows/rw-entry-release-baseset.yml@v4
+    secrets: inherit
+    with:
+      apt-packages: catcodec
```

Comment:
All the Makefile stuff is a pain to read (well I can say unreadable), but I don't see why music pack would use `catcodec`

Reply:
This was already in the workflow, so I took it over. I tried not to make any actually changes in this migration, as that shouldn't be part of this PR, but of a follow-up. There are more weird things in these repositories ... can only do so much at once :D